test/Const.ts

```diff
@@ -22,6 +22,7 @@ describe('Const', () => {
     })
 
     it('mapLeft', () => {
+      // @ts-ignore
       const f = _.mapLeft(S.toUpperCase)
```

Comment:
Fails to infer the input type as `string`.

Reply:
```suggestion
      const f = pipe(S.toUpperCase, _.mapLeft)
```

works, as does

```suggestion
      const f = _.mapLeft<string, string>(S.toUpperCase)
```

Neither feels ideal though.